src/components/DynamicFormContainer/DynamicForm/AttachmentDropzone/AttachmentDropzone.tsx

```diff
@@ -4,7 +4,8 @@ import { FileUploadType } from "../../../../types";
 import { Button } from "../../../UI/Button";
 import { FilesInfo } from "./FilesInfo";
 
-const MAX_FILE_SIZE = 20971520;
+const MAX_FILE_SIZE = 5242880;
+const BYTE_CONVERTION_RATE = 1048576;
```

Comment:
u wanna try this instead https://github.com/visionmedia/bytes.js? this allows u to do conversion both ways (string bytes to bytes and vice versa).

make sure to remove `pretty-bytes` and replace with `bytes` if u using it